### PR TITLE
French translations update

### PR DESCRIPTION
**Pull request by retailcoder**
Includes a few minor tweaks to English strings; added "do not translate" comment to resources not meant to be translated, but might have missed a few.

### CODE REVIEW

File: Rubberduck.Resources/RubberduckUI.resx [hunk 1]
```diff
@@ -1511,7 +1511,7 @@ NOTE: Restart is required for the setting to take effect.</value>
     <comment>{0}: Variable Name</comment>
   </data>
   <data name="MoveCloserToUsage_TargetIsInOtherNonStandardModule" xml:space="preserve">
-    <value>'{0}' is defined neither in the component component it is used not in a standard module. </value>
+    <value>'{0}' is defined neither in the component component it is used nor in a standard module. </value>
```
Comment (bclothier) [inline Rubberduck.Resources/RubberduckUI.resx]: What's a `component component`?

Comment (retailcoder) [reply Rubberduck.Resources/RubberduckUI.resx]: :facepalm:

Comment (bclothier) [reply Rubberduck.Resources/RubberduckUI.resx]: Actually, reading it again, it's still awkward constructed. 

What about this:
```
'{0}' is neither defined nor used in any non-standard modules
```

Comment (retailcoder) [reply Rubberduck.Resources/RubberduckUI.resx]: I wouldn't not wonder if there might not be a way to make that not a bit less, not, ...or is it just me?

File: Rubberduck.Resources/RubberduckUI.resx [hunk 2]
```diff
@@ -1511,11 +1511,11 @@ NOTE: Restart is required for the setting to take effect.</value>
...
     <comment>{0}: Variable Name</comment>
   </data>
   <data name="MoveCloserToUsage_TargetIsNonPrivateInNonStandardModule" xml:space="preserve">
-    <value>'{0}' is a non-private module variable outside a standard module.</value>
+    <value>'{0}' is globally accessible and is in use.</value>
```
Comment (MDoerner) [inline Rubberduck.Resources/RubberduckUI.resx]: That delivers the wrong message. We move globally accessible fields out of standard modules.

**Pull request merged**